corehq/apps/app_manager/templates/app_manager/download_index.html

```diff
@@ -13,28 +13,42 @@
     {% if request|toggle_enabled:"RELEASE_BUILDS_PER_PROFILE" %}
         <h2>App Profiles</h2>
         <table class="table table-condensed">
-            <tr>
-                <th class="col-sm-2">Name</th>
-                <th class="col-sm-2">Languages</th>
-                <th class="col-sm-2">Action</th>
-            </tr>
-            {% for build_profile_id, build_profile in app.build_profiles.items %}
-            <tr>
-                <td class="col-sm-2">{{ build_profile.name }}</td>
-                <td class="col-sm-2">{{ build_profile.langs|join:", " }}</td>
-                <td class="col-sm-2">
-                    {% if build_profile_id in enabled_build_profiles %}
-                        <a href='{% url "toggle_build_profile" app.domain app.id build_profile_id %}?action=disable'>
-                            {% trans 'Revoke restriction' %}
-                        </a>
-                    {% else %}
-                        <a href='{% url "toggle_build_profile" app.domain app.id build_profile_id %}?action=enable'>
-                            {% trans 'Restrict to this version' %}
-                        </a>
-                    {% endif %}
-                </td>
-            </tr>
-            {% endfor %}
+            <thead>
+                <tr>
+                    <th class="col-sm-2">Name</th>
+                    <th class="col-sm-2">Languages</th>
+                    <th class="col-sm-2">Latest Enabled Version</th>
+                    <th class="col-sm-2">Action</th>
+                </tr>
+            </thead>
+            <tbody>
+                {% for build_profile_id, build_profile in app.build_profiles.items %}
+                <tr>
+                    <td class="col-sm-2">{{ build_profile.name }}</td>
+                    <td class="col-sm-2">{{ build_profile.langs|join:", " }}</td>
+                    <td class="col-sm-2">
+                        {% for id, version in latest_enabled_build_profiles.items %}
+                            {% if id == build_profile_id %}
+                                {{ version }}
+                            {% endif %}
+                        {% endfor %}
+                    </td>
+                    <td class="col-sm-2">
+                        {% if build_profile_id in enabled_build_profiles %}
+                            <a class="btn btn-default" href='{% url "toggle_build_profile" app.domain app.id build_profile_id %}?action=disable'>
+                                <i class="fa fa-chain-broken"></i>
+                                {% trans 'Revoke restriction' %}
+                            </a>
+                        {% else %}
+                            <a class="btn btn-default" href='{% url "toggle_build_profile" app.domain app.id build_profile_id %}?action=enable'>
+                                <i class="fa fa-ban"></i>
```

Comment:
nice buttons decorations. 👍

Reply:
🙇‍♀️ so much of my time is spent looking at the fontawesome cheat sheet...